integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/EnvironmentLegacyAuthzGetTest.java

```diff
@@ -29,11 +31,17 @@
     @Inject
     private FreeIpaTestClient freeIpaTestClient;
 
+    @Inject
+    private CloudbreakActor cloudbreakActor;
+
     @Override
     protected void setupTest(TestContext testContext) {
-        useRealUmsUser(testContext, AuthUserKeys.LEGACY_NON_POWER);
-        useRealUmsUser(testContext, AuthUserKeys.LEGACY_POWER);
-        useRealUmsUser(testContext, AuthUserKeys.LEGACY_ACC_ENV_ADMIN);
+        useRealUmsUser(testContext, AuthUserKeys.LEGACY_NON_POWER, REAL_UMS_ENVIRONMENT_KEY, REAL_UMS_ACCOUNT_KEY);
+        useRealUmsUser(testContext, AuthUserKeys.LEGACY_POWER, REAL_UMS_ENVIRONMENT_KEY, REAL_UMS_ACCOUNT_KEY);
+        useRealUmsUser(testContext, AuthUserKeys.LEGACY_ACC_ENV_ADMIN, REAL_UMS_ENVIRONMENT_KEY, REAL_UMS_ACCOUNT_KEY);
+        initializeDefaultBlueprints(testContext);
```

Comment:
Why are these lines needed in the testcase?

Reply:
Originally these are: https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/EnvironmentLegacyAuthzGetTest.java#L34-L36
even all the further actions done by `LEGACY_ACC_ENV_ADMIN`:
- `credentialTestClient.create()`
- `environmentTestClient.create()`
- `freeIpaTestClient.create()`
as you can see.

So this is actually an environment creation test case with `LEGACY_ACC_ENV_ADMIN` user. So I've introduce the related steps in the test setup and, because of the setup part can be done in the `setupTest` with init blueprints and create credential and image catalog with this user...etc.

Reply:
let's discuss this, i am not remembering fully what we have discussed regarding this last time